### PR TITLE
fix: trunk items for policejob

### PR DESCRIPTION
**Pull request by GodyFromDiscord**
## Description

At the moment with the latest qb-inventory & qb-policejob install, the trunk items are not added to PD vehicles trunks, this was due to [this](https://github.com/qbcore-framework/qb-inventory/commit/7561d634936b2ffa893fcfa5dcab9b417a5367c9) pull request, this is due to the exports being server sided and being called from a client side script, I have made it so the exported function is still called in Net event, so it can still be used from client side and fixed PD vehicles trunks not having any items added.

This fixes issue [#445](https://github.com/qbcore-framework/qb-policejob/issues/445) for qb-policejob

## Checklist

- [X] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [X] My code fits the style guidelines.
- [X] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -2261,11 +2265,13 @@ RegisterNetEvent('inventory:server:snowball', function(action)
 		RemoveItem(source, "weapon_snowball")
 	end
 end)
-RegisterNetEvent('inventory:server:addTrunkItems', function()
-	print('inventory:server:addTrunkItems has been deprecated please use exports[\'qb-inventory\']:addTrunkItems(plate, items)')
```
Comment (zwkf) [inline server/main.lua:2265]: Do we want to remove this print?

Comment (GodyFromDiscord) [reply server/main.lua:2266]: Oh yes, my bad I forgot to take that out

Comment (xxUnknown13xx) [reply server/main.lua:2266]: Sorry, So should we NOT remove this then?

**Comment by tom-osborne** [comment]
Merge this. Good PR Gody.

**Review by zwkf — APPROVED** [review]
LGTM

**Review by zwkf — APPROVED** [review]
LGTM

**Pull request merged**